WcaOnRails/app/models/user.rb

```diff
@@ -270,7 +262,7 @@ def self.delegate_status_allows_senior_delegate(delegate_status)
   def not_illegally_demoting_oneself
     about_to_lose_access = !software_team? && !board_member?
     if current_user == self && about_to_lose_access
-      if software_team_was
+      if self.was_team_member?('software')
         errors.add(:admin, "You cannot resign from your role as a software admin team member! Find another person to fire you.")
```

Comment:
Changed it on the model, but the part at user_spec no longer exists...

On Mon, Mar 21, 2016 at 4:52 AM, Jeremy Fleischman <notifications@github.com
> wrote:

> In WcaOnRails/app/models/user.rb
> <https://github.com/cubing/worldcubeassociation.org/pull/390#discussion_r56788963>
> :
>
> > @@ -270,7 +262,7 @@ def self.delegate_status_allows_senior_delegate(delegate_status)
> >    def not_illegally_demoting_oneself
> >      about_to_lose_access = !software_team? && !board_member?
> >      if current_user == self && about_to_lose_access
> > -      if software_team_was
> > +      if self.was_team_member?('software')
> >          errors.add(:admin, "You cannot resign from your role as a software admin team member! Find another person to fire you.")
>
> Oh whoops! It looks like I missed this message in 24dc4e9
> <https://github.com/cubing/worldcubeassociation.org/commit/24dc4e97e1331a289219fa231a0bf42c1844b622>.
> Could you change that from "software admin team" to just "software team"?
> Same for
> https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/spec/models/user_spec.rb#L78
> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/390/files/4cfc692de289d2d4d24e03be7e8db4b02da2a89d#r56788963>
>